src/forms/yup.js

```diff
@@ -19,6 +19,28 @@ import * as yup from 'yup';
 
 const ARRAY_INDEX_REGEX = /\[([^)]+)\]/;
 
+const hasValidTLD = urlString => {
+  try {
+    const TLD_REGEX = /\.[a-z]{2,}$/i;
+    const url = new URL(urlString);
+    const match = url.hostname.match(TLD_REGEX);
+    if (match) {
+      return true;
+    }
+    return false;
```

Comment:
Suggestion: I think this should do the same
```suggestion
    return !!match;
```

Reply:
I recommend `Boolean(match)` instead of `!!match`.